langchain/document_loaders/hugging_face_dataset.py

```diff
@@ -72,13 +73,15 @@ def load(self) -> List[Document]:
             num_proc=self.num_proc,
         )
 
-        docs = [
+        yield from (
```

Comment:
This would be even better if we use streaming datasets https://huggingface.co/docs/datasets/stream

Reply:
if i understood correctly `streaming=True` doesn't download the data locally, but `streaming=False` does?

it sounds like we should pass a `streaming` param to dataset, rather than use `streaming=True` by default. Is that correct?